index.bs

```diff
@@ -8486,12 +8503,11 @@ interfaces. Specifically:
 
 </div>
 
-If the [{{Global}}] or
-[{{PrimaryGlobal}}]
-[=extended attributes=]
-is used on an [=interface=], then:
+If the [{{Global}}] or [{{PrimaryGlobal}}] [=extended attributes=] is
+used on an [=interface=], then:
 
 *   The interface must not define a [=named property setter=].
```

Comment:
Window has a named property setter and IDL defines handling for named property setters on globals.